support-frontend/assets/components/forms/customFields/validated.jsx

```diff
@@ -16,6 +19,15 @@ type Props = PropsForHoc & {
   children?: Option<Node>,
 };
 
+const mainCss = css`
+  .component-input:not([type-radio]) {
+    border-color: ${success['400']};
```

Comment:
It doesn't look as though this is getting applied correctly - do you maybe need to apply the border here as with focus? 
```border: 4px solid ${success[400]};```

I can see why you've applied the validated colour this way, but there's something about this that makes me uncomfortable. It feels not in the spirit of emotion somehow. I'm wondering whether it would be better to pass a prop through to the input field and override the css for the border there.

Reply:
I'm not totally sure what you mean about it getting applied correctly - could you have another look, I've fixed a few things so maybe this no longer applies?

Re the way of applying styling - I think this component will be able to be removed quite soon, I've been discussing with Simon, so I'll leave it as is for now and tidy up later.

Reply:
It was just in the screenshots that you provided, the green outline didn't look like it was 4px, it looked like a 1px outline. The Figma designs have a 4px solid border.